shared/actions/chat.js

```diff
@@ -760,7 +767,7 @@ function * _updateBadging (action: UpdateBadging): SagaGenerator<any, any> {
   const {conversationIDKey} = action.payload
   const conversationStateSelector = (state: TypedState) => state.chat.get('conversationStates', Map()).get(conversationIDKey)
   const conversationState = yield select(conversationStateSelector)
-  if (conversationState && conversationState.messages !== null && conversationState.messages.size > 0) {
+  if (conversationState && conversationState.firstNewMessageID && conversationState.messages !== null && conversationState.messages.size > 0) {
```

Comment:
we only change the badging if we have unread stuff (and therefore the badge would change)